wiki/ccxt.pro.manual.md

```diff
@@ -422,6 +422,27 @@ $loop = \React\EventLoop\Factory::create(); // the event loop goes here ↓
 $exchange = new \ccxt\pro\kucoin(array('loop' => $loop, 'newUpdates': false ));
 ```
 
+## Proxy
```

Comment:
@pcriadoperez Nice we need some docs here 🙂 

The thing is, I think we should emphasize that there are always two options:
- using the trust env (as you described)
- using the option `aiohttp_proxy`

Example:
```Python
exchange = ccxt.binance({
   # your options ..
   aiohttp_proxy = 'proxy.com'
})
```